.github/workflows/check_liveness_after_site_deployment.yml

```diff
@@ -0,0 +1,35 @@
+# .github/workflows/test01.yaml
+name: check_liveness_after_site_deployment.yml
+
+env:
+   AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
+   AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+   AWS_REGION: ${{secrets.AWS_REGION}}
```

Comment:
Instead of using directly AWS , we used intead https://github.com/mongulu-cm/lobembe/issues/44#issuecomment-1311100835 .

It's currently done on deploy_push.yml with https://github.com/mongulu-cm/mtchoun-mouh/blob/40bf4625e79652b750c1a8daf87b8874fe6ea58c/.github/workflows/close_pr.yml#LL9-L11C21 + https://github.com/mongulu-cm/mtchoun-mouh/blob/40bf4625e79652b750c1a8daf87b8874fe6ea58c/.github/workflows/close_pr.yml#LL20-L25C35

Reply:
great ! I will check it .